### ISSUE TITLE
Icons about the calendar not showing correctly

### ISSUE THREAD

**Issue by hxy0229** [monim67/django-bootstrap-datepicker-plus#16]
**Describe the problem**
A clear and concise description of what the scenario is.

**To Reproduce**

**Expected behavior**

**Console errors**
- No error on django console.
- No error on JavaScript console.

**Paste-bin**
The page is showing up but the date-picker calendar is not. The paste-bin link
of the HTML source of the page is http://...

**Setup Information (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Browser version [e.g. 22]
 - Python version [e.g. 3.7]
 - Django version [e.g. 2.1]
 - Bootstrap version [e.g. 3/4]
 - jQuery version [e.g. 1.7.1]

[x] I have followed the [configuration instructions][configuration_page] and checked out the
[common error troubleshooting][troubleshooting_page] page.

[configuration_page]: https://monim67.github.io/django-bootstrap-datepicker-plus/configure/
[troubleshooting_page]: https://django-bootstrap-datepicker-plus.readthedocs.io/en/latest/Troubleshooting.html

**Comment by hxy0229** [comment]
![_20190123113448](https://user-images.githubusercontent.com/31153144/51581409-e8f0c680-1f02-11e9-8d11-8bad8ebdae17.jpg)
The icons(buttons) on the calendar are not showing correctly.

**Comment by monim67** [comment]
Please provide your setup information (see the first comment here). Which configuration process are you following?

**Comment by OdinCosta** [comment]
Hi monim67,
I have the exact same issue. I'm filling up the information hxy0229 didn't for my problem. I tried a few solutions but none of them work. Any help is appreciated, thank you.

Describe the problem
Calendar works fine but buttons don't show. Instead, empty squared boxes appear.

To Reproduce
Steps to reproduce the behavior:
- Run the server and open the calendar, you will see the empty boxes. Same as hxy0229 uploaded screenshot.

Expected behavior
- Buttons showing same way as in the examples.

Console errors
- No error on django console.
- No error on JavaScript console.

Paste-bin
- What is this?

Setup Information (please complete the following information):

OS: Windows 10
Browser: Chrome
Browser version: 77.0.3865.75
Python version: 3.7.3
Django version: (2, 2, 1, 'final', 0)
Bootstrap version: 3
jQuery version: How can I check that? I followed your tutorials I assume I have the current one.
[x] I have followed the configuration instructions and checked out the
common error troubleshooting page.

Linked I followed:
- https://monim67.github.io/django-bootstrap-datepicker-plus/configure/
- https://django-bootstrap-datepicker-plus.readthedocs.io/en/latest/Walkthrough.html

**Comment by OdinCosta** [comment]
See the three missing icons:
![image](https://user-images.githubusercontent.com/26311180/67676425-5db4cd80-f979-11e9-9509-5ab59014134d.png)

**Comment by OdinCosta** [comment]
@monim67 
Let me know if you need any more info pls!
Thank you very much!

**Comment by monim67** [comment]
Is there any way I can check it out myself? You sure the font files are loading properly?

**Comment by OdinCosta** [comment]
@monim67 I think I have isolated the root of the problem.
Seems to be conflicting with my master template.

- If I remove {% extends 'Maturities/base.html' %}
- The icon appears but then I lose all my formatting

How can I have both? - I configured both my master and page following:

- https://monim67.github.io/django-bootstrap-datepicker-plus/configure/
- Options: Bootstrap3, Include using django-bootstrap3 tags, Include Jquery from CDN, I have a master layout template.

If I can't have both, I assume I will write all the style on my page html.
BTW, this widget is awesome. Nice work.
